src/brogue/Items.c

```diff
@@ -1689,7 +1689,7 @@ boolean isVowelish(char *theChar) {
     }
     char str[30];
```

Comment:
Strictly this could be str[29] but if we leave it as str[30] it will be null-terminated in most cases.
Another option would be to add str[29] = '\0' to ensure null termination.
Or just leave as is.

Reply:
We should definitely ensure strings are null-terminated, I think that is the real issue with this function. Second option sounds good

Reply:
Okay, reimplemented. Interestingly now you can switch the first 30 for 29 and the last 29 for 30 with no effect! But I think this implementation is best.